<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NombreComplexe</class>
 <widget class="QDialog" name="NombreComplexe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NombreComplexe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 244, 244);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Partie r√©elle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="reelInput">
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>311</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Partie Imaginaire : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="imgInput">
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>211</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="saveBtn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Enregistrer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitBtn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Module</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="mdlInput">
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TP-1</string>
   </property>
  </widget>
  <zorder>textEdit_2</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>label</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
  <zorder>label_2</zorder>
  <zorder>horizontalLayoutWidget_4</zorder>
  <zorder>label_4</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>NombreComplexe</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
